recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc

```diff
@@ -335,7 +330,7 @@ FILES:libegl-imx-dev = "${includedir}/EGL ${includedir}/KHR ${libdir}/pkgconfig/
 INSANE_SKIP:libegl-imx += "dev-so"
 
 FILES:libgal-imx = "${libdir}/libGAL${SOLIBS} ${libdir}/libGAL_egl${SOLIBS}"
-FILES:libgal-imx-dev = "${includedir}/HAL"
+FILES:libgal-imx-dev:remove = "${includedir}/HAL"
```

Comment:
Sorry, to be clear, the original line here just needed to be removed. Keep in mind that what you find in `meta-imx` is overriding what is in `meta-freescale`, so the integration is not always a direct copy.